deployment/cloud-run/debug_errorhandler_detailed.py

```
@@ -1,79 +1,82 @@
 # Let's check if there are any global variables that might be interfering
-print(f"\n🔍 Checking for global variable conflicts...")
+print("\n🔍 Checking for global variable conflicts...")
 print(f"Built-in errorhandler: {getattr(__builtins__, 'errorhandler', 'Not found')}")
 print(f"Global errorhandler: {globals().get('errorhandler', 'Not found')}")
 
 # Let's check if there's a version issue
 try:
     import flask_restx
+
     print(f"\n🔍 Flask-RESTX version: {flask_restx.__version__}")
     print(f"Flask version: {flask.__version__}")
```

Ruff (0.13.1)

Undefined name `flask`

(F821)